apps/alphabet_mobile/ios/alphabet_mobile/main.m

```diff
@@ -0,0 +1,10 @@
+#import <UIKit/UIKit.h>
+
+#import "AppDelegate.h"
+
+int main(int argc, char *argv[])
```

Comment:
Is this C? I think you win the polyglot of the month award with this one. This is like the 5th language I've seen now.